.teamcity/common/performance-test-extensions.kt

```diff
@@ -36,6 +36,9 @@ fun BuildType.applyPerformanceTestSettings(os: Os = Os.LINUX, timeout: Int = 30)
         param("env.JAVA_HOME", os.buildJavaHome())
         param("env.BUILD_BRANCH", "%teamcity.build.branch%")
         param("performance.db.username", "tcagent")
+        if (os == Os.WINDOWS) {
+            param("env.JPROFILER_HOME", "C:\\Program Files\\jprofiler\\jprofiler11.1.4")
+        }
```

Comment:
This looks out of place here. Might be just me.

Reply:
I can move this to `Os`.

Reply:
Moved.